dlme_airflow/drivers/oai_xml.py

```diff
@@ -60,10 +63,26 @@ def _open_set(self):
                 )
             ]
         else:
-            oai_records = self._collection.ListRecords(
-                set=self.set, metadataPrefix=self.metadata_prefix, ignore_deleted=True
+            opts = {
+                "set": self.set,
+                "metadataPrefix": self.metadata_prefix,
+                "ignore_deleted": True,
+            }
+
+            if self.last_harvest_start_date and not self.full_harvest:
+                # need to use opts dict since "from" is a reserved word
+                opts["from"] = self.last_harvest_start_date.strftime("%Y-%m-%d")
+
+            logging.info(
+                "Harvesting %s with OAI ListRecords: %s", self.collection_url, opts
```

Comment:
It looks a bit weird to pass `opts` in instead of the usual parameters, but Sickle chose to hind really closely to OAI-PMH parameters and `from` is a reserved word in Python.

Reply:
Yeah - I found this out when doing the manual work to get QNL by years, it's in the Sickle documentation that you have to pass opts this way or an error is thrown. :+1: